src/MissionCreator.sol

```diff
@@ -142,6 +147,7 @@ contract MissionCreator is Ownable, IERC721Receiver {
             lockedUntil: 0, // The split is not locked, meaning the project owner can remove it or change it at any time.
             hook: IJBSplitHook(address(0)) // Not used.
         });
+        // project token split
         rulesetConfigurations[0].splitGroups[1].splits[1] = JBSplit({
             percent: 300_000_000, // 30%, out of 1_000_000_000
             projectId: 420, // The projectId of the project to send the split to.
```

Comment:
How will this be determined? Should go to their multisig address right?

Reply:
yes this is the multisig.